compiler+runtime/src/cpp/jank/runtime/context.cpp

```diff
@@ -480,7 +485,7 @@ namespace jank::runtime
     native_persistent_string resolved_ns{ ns };
     if(!resolved)
     {
-      /* The ns will be an ns alias. */
+      /* The ns will be an ns or alias. */
```

Comment:
If it's not resolved, it'll be an alias. There is no `or` here. `::foo/bar` is an unresolved keyword and `foo` *must* be an alias.

From a clj repl:

```
user=> ::clojure.core/foo
Syntax error reading source at (REPL:3:0).
Invalid token: ::clojure.core/foo

user=> :clojure.core/foo
:clojure.core/foo
```

Reply:
Hm, the error is inside out here. The note should be the message, while the note should be "Found here". I'll fix that on my end, since it's not related to your PR.